.babelrc.js

```diff
@@ -1,5 +1,7 @@
 "use strict";
 
+const relative = require("require-relative");
+
 // Blame Logan for this.
```

Comment:
> Blame Logan for this.

Let's remove this?

Reply:
https://github.com/istanbuljs/istanbuljs/issues/92 is still open, so I think that that function is still needed.

Reply:
I meant the comment. We can't blame Logan for this